src/rgb_packet_processor.cpp

```diff
@@ -50,21 +53,31 @@ void RgbPacketProcessor::setFrameListener(libfreenect2::FrameListener *listener)
 }
 
 DumpRgbPacketProcessor::DumpRgbPacketProcessor()
+  : frame(NULL)
 {
 }
 
 DumpRgbPacketProcessor::~DumpRgbPacketProcessor()
 {
+  delete frame;
```

Comment:
yeah, I figured that out ;)  I deleted the member variable entirely, its not needed.